src/components/shared/page.tsx

```diff
@@ -80,26 +82,27 @@ function getArticleSubtype(capi: Content): FunctionComponent<ArticleProps> {
         return ImmersiveArticle;
     }
   
-    return Article;
+    return ArticleComponent;
```

Comment:
If Default is the term we will use shouldn't that be used here too? e.g. `DefaultArticle` to be consistent with `OpinionArticle`, `LiveblogArticle` ...

All of the `article` components live inside a `news` folder, should that now be a default folder?

Reply:
I think so. An alternative name could also be `Standard`, but that could be somewhat ambiguous...

Reply:
I thought about `Standard` too and it looks like dcr are using that term